.github/workflows/create-tag.yml

```diff
@@ -1,16 +1,18 @@
 name: db-syncer -Create Tag
 
 on:
-  workflow_dispatch:
+  workflow_call:
     inputs:
       ref:
-        description: "The branch to checkout when creating the image."
-        required: true
-        default: "release_pipeline"
+        type: string
+        default: main
+      repo: 
+        type: string
+        default: main
```

Comment:
Do we want to set a default for `ref` and `repo`? With this being a release, I suspect these fields are always required.